### PR TITLE
Support last_response_count in API responses

### PR DESCRIPTION
**Pull request by mulby**
@dsjen - I realized that I had been seeing this error when I was developing the analytics devstack, so I was able to rapidly develop a patch using that environment.

### CODE REVIEW

**Comment by dsjen** [comment]
Hurray!!!  :baby_chick:  We should add a test to cover this code path too--probably around https://github.com/edx/edx-analytics-dashboard/blob/master/analytics_dashboard/courses/tests/test_presenters.py#L442.

**Comment by mulby** [comment]
@dsjen tests updated

File: analytics_dashboard/courses/presenters/performance.py [hunk 1]
```diff
@@ -153,6 +153,12 @@ def _build_questions(self, answer_distributions):
     def _build_answer_distribution(self, api_response, problem_part_id):
         """ Filter for this problem part and sort descending order. """
         answer_distributions = [i for i in api_response if i['part_id'] == problem_part_id]
+        for answer_dist in answer_distributions:
+            # First and last response counts were added, we can handle both types of API responses at the moment.
+            # If just the count is specified it is assumed to be the last response count.
+            count = answer_dist.get('last_response_count')
+            if count is not None:
+                answer_dist['count'] = count
```
Comment (brianhw) [inline analytics_dashboard/courses/presenters/performance.py:162]: I assume this is a temporary modification.  I would assume going forward we would instead want this to actually look for 'last_response_count' in places that previously accessed count, and have it map 'count' to 'last_response_count' in cases where we have the former and not the latter.

Comment (dsjen) [inline analytics_dashboard/courses/presenters/performance.py:162]: Good point.  There is some technical debt here in order to support the upcoming deployment without downtime.  I've captured the debt in AN-5761.  @mulby, would you please add a TODO?

Comment (mulby) [inline analytics_dashboard/courses/presenters/performance.py:162]: Yeah I thought about that a bit. It seems like the natural thing to change once we actually support `first_response_count`. I can add a TODO here.

**Comment by mulby** [comment]
@dsjen tests are passing now

File: analytics_dashboard/courses/tests/test_presenters.py [hunk 2]
```diff
@@ -428,18 +429,43 @@ def test_get_education(self, mock_education):
...
 
+PERFORMER_PRESENTER_COURSE_ID = 'edX/DemoX/Demo_Course'
+
+
+class ListWithName(list):
+    pass
+
+
+def annotated(l, name):
+    alist = ListWithName(l)
```
Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:440]: Is this a way of naming the dataset?

**Comment by dsjen** [comment]
LGTM! :+1:

File: analytics_dashboard/courses/tests/test_presenters.py (hunk 2 above)
Comment (mulby) [inline analytics_dashboard/courses/tests/test_presenters.py:440]: Yup - otherwise it autogenerates a massive name that is very difficult to read!

Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:440]: Very interesting!

**Pull request merged**